requirements/nightly.txt

```diff
@@ -66,37 +74,75 @@ mpltoolbox==24.5.1
     # via scippneutron
 networkx==3.4.2
     # via cyclebane
+numpy==2.2.1
+    # via
+    #   -r nightly.in
+    #   contourpy
+    #   h5py
+    #   ipydatawidgets
+    #   matplotlib
+    #   mpltoolbox
+    #   pandas
+    #   pythreejs
+    #   scipp
+    #   scippneutron
+    #   scipy
+packaging==24.2
+    # via
+    #   dask
+    #   matplotlib
+    #   pooch
+    #   pytest
+pandas==2.2.3
+    # via -r nightly.in
 parso==0.8.4
     # via jedi
 partd==1.4.2
     # via dask
 pexpect==4.9.0
     # via ipython
-pillow==11.0.0
+pillow==11.1.0
     # via matplotlib
+platformdirs==4.3.6
+    # via pooch
 plopp @ git+https://github.com/scipp/plopp@main
     # via
     #   -r nightly.in
     #   scippneutron
+pluggy==1.5.0
+    # via pytest
+pooch==1.8.2
+    # via -r nightly.in
 prompt-toolkit==3.0.48
     # via ipython
 ptyprocess==0.7.0
     # via pexpect
 pure-eval==0.2.3
     # via stack-data
-pygments==2.18.0
+pygments==2.19.1
     # via ipython
-pyparsing==3.2.0
+pyparsing==3.2.1
     # via matplotlib
+pytest==8.3.4
+    # via -r nightly.in
+python-dateutil==2.9.0.post0
+    # via
+    #   matplotlib
+    #   pandas
+    #   scippnexus
 pythreejs==2.4.2
     # via -r nightly.in
+pytz==2024.2
+    # via pandas
 pyyaml==6.0.2
     # via dask
+requests==2.32.3
+    # via pooch
 sciline @ git+https://github.com/scipp/sciline@main
     # via
     #   -r nightly.in
     #   essreduce
-scipp @ https://github.com/scipp/scipp/releases/download/nightly/scipp-nightly-cp310-cp310-manylinux_2_17_x86_64.manylinux2014_x86_64.whl
+scipp==24.11.3.dev27+g47efd60e3
```

Comment:
This either needs to be fixed while building or uploading wheels to the index. It should ideally be a stable version number like `scipp==24.11.3.dev0` or the nightly requirements shouldn't be locked with a pinned requirements file.